app/javascript/packs/base.jsx

```diff
@@ -4,6 +4,40 @@ import {
   getInstantClick,
   initializeTouchDevice,
 } from '../topNavigation/utilities';
+import { fetchSearch } from '../utilities/search';
+
+window.initializeMentionAutocomplete = async (element) => {
```

Comment:
Don't want to just fill the window with functions, but we need to access this from erb files so I think it has to be here

Reply:
Is there a scenario where we put `window.initializeMentionAutocomplete` in `/packs/articlePage.jsx` or is it limited by the other code sitting in `base.js.erb`?

I wonder if there's a way to import the component only in the pages where we have comment forms

Reply:
the Preact imports only happen when `initializeMentionAutocomplete` is called, so it'll only be imported into pages with comment forms 👍